src/components/side_panel/pivot/pivot_side_panel/pivot_side_panel_store.ts

```diff
@@ -195,6 +195,7 @@ export class PivotSidePanelStore extends SpreadsheetStore {
         format: measure.format,
         display: measure.display,
       })),
+      sortedCol: this.shouldRemoveSortedColumn(definition) ? definition.sortedCol : undefined,
```

Comment:
The name of `shouldRemoveSortedColumn` is a bit misleading. This line reads as "If the sorted columns should be removed, then don't remove it"